tests/e2e/test_get_block_number.py

```diff
@@ -18,7 +18,7 @@ def setUp(self) -> None:
 
     def test_get_block_number(self):
         self.fetcher.period = AccountingPeriod("1970-01-01")  # Before Time
-        self.assertEqual(self.fetcher.get_block_interval(), ("0", "0"))
+        self.assertEqual(self.fetcher.get_block_interval(), ("None", "None"))
```

Comment:
Dune changed something about block fetching recently and what was zero before (the first block existed) is now None.